skymp5-server/cpp/server_guest_lib/PartOne.cpp

```diff
@@ -548,15 +548,25 @@ void PartOne::Init()
     uint32_t worldOrCell =
       emitter->GetCellOrWorld().ToFormId(worldState.espmFiles);
 
+    // See 'perf: improve game framerate #1186'
+    // Client needs to know if it is DOOR or not
+    const char* baseRecordTypePrefix = "";
+    std::string baseRecordType;
+    if (const std::string& baseType = emitter->GetBaseType();
+        baseType == "DOOR") {
+      baseRecordTypePrefix = R"(, "baseRecordType": )";
```

Comment:
We could probably not send spaces here at all

Reply:
reasonable, but I think it's for another PR